_posts/2022-02-11-helm-multiple-worker-groups.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: How to run different worker types with the Dask Helm Chart
+author: Matthew Murray (NVIDIA)
+tags: [Kubernetes, Helm]
+theme: twitter
+---
+
+## Introduction
+Today, we’ll learn how to deploy Dask on a Kubernetes cluster with the Dask Helm Chart and then run and scale different worker types with annotations.
+
+### What is the Dask Helm Chart?
+The Dask Helm Chart is a convenient way of deploying Dask using Helm, a package manager for Kubernetes applications. After deploying Dask with the Dask Helm Chart, we can connect to our HelmCluster and begin scaling out workers.
+
+### What is Dask Kubernetes?
+Dask Kubernetes allows you to deploy and manage your Dask deployment on a Kubernetes cluster. The Dask Kubernetes Python package has a HelmCluster class (among other things) that will enable you to manage your cluster from Python. In this tutorial, we will use the HelmCluster.
+
+### Prerequisites:
+- To have Helm installed and be able to run the ‘helm’ commands
+- To have a running Kubernetes cluster. It doesn’t matter whether you're running Kubernetes locally using [MiniKube](https://minikube.sigs.k8s.io/docs/) or [Kind](https://kind.sigs.k8s.io/) or you're using a cloud provider like AWS or GCP. But your cluster will need to have access to [GPU nodes](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/) to run GPU workers. You’ll also need to install [RAPIDS](https://rapids.ai/) to run the GPU worker example.
+- [Optional] To have [kubectl](https://kubernetes.io/docs/tasks/tools/) installed.
+That’s it, let’s get started!
+
+## Install Dask Kubernetes
+From the [documentation](https://kubernetes.dask.org/en/latest/installing.html),
+```console
+pip install dask-kubernetes --upgrade
+```
+or
+```console
+conda install dask-kubernetes -c conda-forge
+```
+
+## Install the Dask Helm Chart
+First, deploy Dask on Kubernetes with Helm:
+```console
+helm repo add dask https://helm.dask.org/
+helm repo update
+helm install my-dask dask/dask
+```
+
+Now you should have Dask running on your Kubernetes cluster. If you have kubectl installed, you can run `kubectl get all -n default`
+<img src="/images/default-dask-cluster.png" alt="Default Dask Cluster Installed with Helm" width="661" height="373">
+
+You can see that we’ve created a few resources! The main thing to know is that we start with three dask workers. Remember, it doesn’t matter if you don't have kubectl installed because we’ll do something similar with HelmCluster 
+
+## Add GPU worker group to our Dask Deployment
+To create some GPU workers, we’ll need to change the default values in the Dask Helm Chart. To do this, we can create a copy of the current [`values.yaml`](https://github.com/dask/helm-chart/blob/main/dask/values.yaml), update it to add a GPU worker group and then update our helm deployment.  
```

Comment:
Why do we need to create the default values? It may not be clear to the user. It would be good to explain that the helm chart has some default values that it uses out of the box, but now that we want to add some configuration we need to create our own `values.yaml` file.